examples/notebooks/langchain_evaluation.ipynb

```diff
@@ -0,0 +1,719 @@
+                            "   'reference': {'answer': 'Montresor is insulted by Lenora and seeks revenge.'}}}}"
+                        ]
+                    },
+                    "execution_count": 60,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "from langsmith import Client\n",
+                "from langchain.evaluation import EvaluatorType\n",
+                "from langchain.smith import RunEvalConfig, run_on_dataset\n",
+                "\n",
+                "evaluation_config = RunEvalConfig(\n",
+                "    # LangChain offers several QA Evaluator types\n",
+                "    evaluators=[\n",
+                "        \"qa\",\n",
+                "        \"context_qa\",\n",
+                "        \"cot_qa\",\n",
+                "    ],\n",
+                "    prediction_key=\"result\",\n",
+                ")\n",
+                "\n",
+                "client = Client()\n",
+                "run_on_dataset(\n",
+                "    dataset_name=dataset_name,\n",
+                "    llm_or_chain_factory=create_qa_chain(llm=model, vstore=vstore),\n",
+                "    client=client,\n",
+                "    evaluation=evaluation_config,\n",
+                "    verbose=True,\n",
```

Comment:
maybe re-run with this set `False`, to significantly reduce the output in the notebook?